build.go

```diff
@@ -86,7 +86,6 @@ var targets = map[string]target{
 	"all": {
 		// Only valid for the "build" and "install" commands as it lacks all
 		// the archive creation stuff. buildPkgs gets filled out in init()
-		tags: []string{"purego"},
```

Comment:
What's this `purego` change about?

Reply:
Cleanup. We used this tag in 2016 when we depended on github.com/cznic/ql for the discovery server, and the tag indicated we didn't want some C dependencies that would fail to cross build. That thing is _long_ gone from anything.